docs/cni/kubernetes/AWS.md

```diff
@@ -176,7 +176,12 @@ aws ec2 run-instances \
 #  Include the subnet param above if using a non-default VPC
 ```
 
-Deploy at least one worker node using the following command:
+Now, deploy at least one worker node.
+
+First, edit `cloud-config/node-config.yaml` and replace all instances of `kubernetes-master` with your Master's private
```

Comment:
Maybe do this with a sed command?  e.g.
`sed -i 's/kubernetes-master/ip-172-35-0-135.eu-west-1.compute.internal/g' cloud-config/node-config.yaml`